src/createAsyncThunk.ts

```diff
@@ -87,63 +86,94 @@ export function createAsyncThunk<
   const rejected = createAction(
     type + '/rejected',
     (error: Error, requestId: string, args: ActionParams) => {
+      if (error.name === 'AbortError') {
```

Comment:
Any chance we can collapse these two cases to shorten it a bit?

Reply:
Better?